framework/src/variables/MooseVariableBase.C

```diff
@@ -56,3 +56,9 @@ MooseVariableBase::order() const
 {
   return _fe_type.order;
 }
+
+bool
+MooseVariableBase::isVector() const
+{
+  return (_fe_type.family == LAGRANGE_VEC || _fe_type.family == NEDELEC_ONE);
```

Comment:
Worried about needing to remember to update this if stuff changes.

What do you think about making this virtual - and letting the MooseVariable derived types implement it to be true/false.

@lindsayad do you have a comment on that?

Reply:
It would be straightforward to implement it in the derived classes. @jwpeterson and I talked briefly about this. One implementation would be:
```c++
template <typename T>
bool
MooseVariableFEImpl<T>::isVector() const
{
  return std::is_same<T, RealVectorValue>::value;
}
```

Reply:
Yeah I know this isn't ideal.  We could add template specializations of `FE<Dim,Family>::is_vector()` that returns true/false at the libmesh level, but I don't think that would help since we really just have an `FEType` at the `MooseVariableBase` level...